src/sorts.tsx

```diff
@@ -64,6 +64,18 @@ const SORTS: Sort[] = [
     name: "Callsign",
     impl: (a, b) => a.callsign.localeCompare(b.callsign),
   },
+  {
+    name: "Type",
```

Comment:
can definitely add a tag to this option to separate repeater and person in the column view